posthog/hogql/functions/mapping.py

```diff
@@ -179,6 +179,7 @@ class HogQLFunctionMeta:
     "toJSONString": HogQLFunctionMeta("toJSONString", 1, 1),
     "parseDateTime": HogQLFunctionMeta("parseDateTimeOrNull", 2, 3, tz_aware=True),
     "parseDateTimeBestEffort": HogQLFunctionMeta("parseDateTime64BestEffortOrNull", 1, 2, tz_aware=True),
+    "_accurateCastOrNull": HogQLFunctionMeta("accurateCastOrNull", 2, 2),
```

Comment:
`accurateCastOrNull` is a great find! Let's actually turn all of `toInt64OrNull`, `toFloat64OrNull`, `toDecimal64OrNull`, `toDateOrNull`, `toUUIDOrNull` into their `accurateCastOrNull` equivalents. This way _all_ queries will get this improvement for free

Reply:
Yeah - I'll make a new PR. From your list I think we both agree that I should not touch the date/time functions